ChelseasFieldJournal/src/GUI/GuiComposerHelpers.cs

```diff
@@ -0,0 +1,836 @@
+﻿using Cairo;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Vintagestory.API.Client;
+using Vintagestory.API.Common;
+
+namespace ChelseasFieldJournal
+{
+    public static class GuiComposerHelpers
+    {
+
+        public static GuiElementColorListPicker GetColorListPicker(this GuiComposer composer, string key)
+        {
+            return (GuiElementColorListPicker)composer.GetElement(key);
+        }
+
+        public static void ColorListPickerSetValue(this GuiComposer composer, string key, int selectedIndex)
+        {
+            int i = 0;
+            GuiElementColorListPicker btn;
+            while ((btn = composer.GetColorListPicker(key + "-" + i.ToString())) != null)
+            {
+                btn.SetValue(i == selectedIndex);
+                i++;
+            }
+        }
+        public static GuiComposer AddStaticImage(this GuiComposer composer, AssetLocation imageAsset, ElementBounds bounds, Operator blendMode = Operator.Over)
+        {
+            bool flag = !composer.Composed;
+            if (flag)
```

Comment:
Replace with `if(!composer.Composed)` instead to keep it standard like the other methods.